.github/workflows/ci.yml

```diff
@@ -152,7 +152,7 @@ jobs:
       - name: Build
         run: |
           ./autogen.sh
-          PKG_CONFIG_PATH=/usr/local/opt/curl/lib/pkgconfig:/usr/local/opt/openssl/lib/pkgconfig ./configure CXXFLAGS='-std=c++03 -DS3FS_PTHREAD_ERRORCHECK=1'
+          PKG_CONFIG_PATH=/usr/local/opt/curl/lib/pkgconfig:/usr/local/opt/openssl/lib/pkgconfig ./configure CXXFLAGS='-std=c++11 -DS3FS_PTHREAD_ERRORCHECK=1'
```

Comment:
Previously we supported Ubuntu 16.04 and RHEL 6 to a degree #1354 which only supported C++03.  We have talked about C++11 in the context of using the AWS SDK #1068.  I have no objection to this change but want to make sure that it is a conscious one motivated by the json library requirement.